docs/integration_resources/software/houdini.md

```diff
@@ -0,0 +1,28 @@
+# Houdini
+
+## Running Shift in Houdini
+
+Shift provides a `hshift` utility script to properly bind the Shift UI to Houdini. In order to run and open Shift in Houdini, execute the following code block inside the Houdini *Python Script Editor*:
+
+```python
+import sys
+sys.path.append("<path_to_your_shift_folder>")
+sys.path.append("<path_to_your_shift_folder>/thirdparty/python/Lib/site-packages")
+
+from shift.tools import hshift
+hshift.show()
+```
+
+## Python Interpreter Setup
+In Shift it is possible to launch the execution of a workflow via an application's Python interpreter with the [WorkflowProcess](../reference/nodes/workflow/#workflowProcess-node) node. To achieve this for Houdini, set path to its Python interpreter in your environment variables:
+
+`SHIFT_PROCESS_HOUDINI : path to Houdini's Python interpreter (hython)`

```

Comment:
Let's change this to a cmd code example for Windows:

```suggestion
For example in windows you can set:
```python
set SHIFT_PROCESS_HOUDINI=<path_to_Houdini_Python_interpreter_(hython)>`
```
```

Reply:
What if the user is in Linux? Should both examples be added? 
Also, a user would usually set the environment variables directly to their machine through the environment variables window (at least that's how I do it). Do you think is necessary to go through how to set the environment variable?

Reply:
Well, it is not necessary to specify how to set it but we should align all on how we want to tell the user what to set. The way you used here I find a bit confusing because "variable": "something" does not correspond to any legit way to set it neither hints to a clean and direct interpretation of what it means. I am thinking imagining the reader not being accustomed at all with these types of concepts.

Does anyone @Inbibo/shift have an idea for a suggestion?

Reply:
If we want to put an example we should put an example for both Linux and Windows

Reply:
I think it not required to explain how to set them in Windows or Linux. The user target of Shift is a Developer that will know how to configure the environment. In other DCCs like Nuke:
![image](https://github.com/user-attachments/assets/91742536-1593-48e3-9204-77777adee818)
or Maya:
![image](https://github.com/user-attachments/assets/caf1b2b1-4ec5-433c-80b8-9af4172f63b0)

they user tables or just headers to specify the variables. But it's weird to found so detailed documentation to explain how to set up a variable in Windows, Linux, Mac,...

Reply:
makes sense then. Maybe we can just put the variable name in bold text like in the maya docs example from Jorge and say what it should point to.

Reply:
In case you are not aware, Icreated a card to write a page with all the environment variables available for Shift. The idea is that this would be like a table guide for all the environment variables, like the one from the page pictures Jorge attached.
https://github.com/Inbibo/shift_docs/issues/20

Based on your last comment, in your request for this page, you mean to format it this way:
![image](https://github.com/user-attachments/assets/1eac0c3e-3950-4a29-80c6-23c6af78d9f4)

Instead of this way:
![image](https://github.com/user-attachments/assets/c63fafb8-d7f9-4f72-9483-698e9ee4f768)

??

Reply:
For me it is much better. Can we get an opinion from @Inbibo/shift so we can reformat all of the env var mentions in the documentation to this?

Reply:
That way of formatting is alright for me too

Reply:
For the moment, I changed the format to the first option

Reply:
I am ok with resolving this if everybody else is ok with it. Will take care of updating to the same format in my PRs. Thank you @juliavisserinbibo !